### ISSUE TITLE
Parme Glue plugin doesn't include all files in name type map

### ISSUE THREAD

**Issue by profexorgeek** [KallDrexx/ParME#55]
I have these files in Glue:
![image](https://user-images.githubusercontent.com/711100/145906408-bd99cf7f-2614-4705-9dff-733556a2d370.png)

The most recently added file is `impactMagic.emlogic` but this file is not being added to the generated Dictionary:
```csharp
private readonly Dictionary<string, Type> _nameTypeMap = new Dictionary<string, Type>()
{
    {"AreaFire", typeof(AreaFireEmitterLogic)},
    {"AreaFireflies", typeof(AreaFirefliesEmitterLogic)},
    {"AreaLeaves", typeof(AreaLeavesEmitterLogic)},
    {"AreaPortal", typeof(AreaPortalEmitterLogic)},
    {"ImpactSmoke", typeof(ImpactSmokeEmitterLogic)},
    {"ImpactSmokeLarge", typeof(ImpactSmokeLargeEmitterLogic)},
    {"PointFire", typeof(PointFireEmitterLogic)},
    {"SplatterCoins", typeof(SplatterCoinsEmitterLogic)},
    {"SplatterSmoke", typeof(SplatterSmokeEmitterLogic)},
    {"SplatterStone", typeof(SplatterStoneEmitterLogic)},
    {"SplatterWater", typeof(SplatterWaterEmitterLogic)},
    {"SplatterWood", typeof(SplatterWoodEmitterLogic)},
    {"TrailFootprints", typeof(TrailFootprintsEmitterLogic)},
    {"TrailSmokeBlue", typeof(TrailSmokeBlueEmitterLogic)},

};
```

I tested this by creating and adding a new file. In all cases it seems that the most recently added .emlogic file is not included in the Glue-generated `Dictionary`.

**Comment by KallDrexx** [comment]
Fixed in 86df337ece25106e069bbf8ba13a3389a17a0083

**Comment by KallDrexx** [comment]
Released v10.7 with fix
